pkg/hyper/streaming.go

```diff
@@ -51,17 +72,17 @@ func (h *Runtime) ExecSync(rawContainerID string, cmd []string, timeout int64) (
 	return stdoutBuffer.Bytes(), stderrBuffer.Bytes(), exitCode, nil
 }
 
-// Exec execute a command in the container.
-func (h *Runtime) Exec(rawContainerID string, cmd []string, tty bool, stdin io.Reader, stdout, stderr io.WriteCloser) error {
-	return fmt.Errorf("Not implemented")
+// Exec prepares a streaming endpoint to execute a command in the container.
+func (h *Runtime) Exec(req *kubeapi.ExecRequest) (*kubeapi.ExecResponse, error) {
+	return nil, fmt.Errorf("Not implemented")
 }
 
 // Attach prepares a streaming endpoint to attach to a running container.
-func (h *Runtime) Attach() error {
-	return fmt.Errorf("Not implemented")
+func (h *Runtime) Attach(req *kubeapi.AttachRequest) (*kubeapi.AttachResponse, error) {
+	return nil, fmt.Errorf("Not implemented")
 }
 
 // PortForward prepares a streaming endpoint to forward ports from a PodSandbox.
-func (h *Runtime) PortForward() error {
-	return fmt.Errorf("Not implemented")
+func (h *Runtime) PortForward(req *kubeapi.PortForwardRequest) (*kubeapi.PortForwardResponse, error) {
+	return nil, fmt.Errorf("Not implemented")
```

Comment:
Also here, refer https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/server/streaming/server.go#L169.

Need add a todo since it is not implemented yet.